Submodule learnRL/pytorch_car_caring contains untracked content
Submodule learnRL/pytorch_car_caring contains modified content
diff --git a/learnRL/pytorch_car_caring/README.md b/learnRL/pytorch_car_caring/README.md
index 411ff58..232b8ab 100644
--- a/learnRL/pytorch_car_caring/README.md
+++ b/learnRL/pytorch_car_caring/README.md
@@ -1,3 +1,4 @@
+
 # Car Racing with PyTorch
 Solving the car racing problem in OpenAI Gym using Proximal Policy Optimization (PPO). This problem has a real physical engine in the back end. You can achieve real racing actions in the environment, like drifting. 
 
diff --git a/learnStableBaselines/7_wandb/1_wandb.py b/learnStableBaselines/7_wandb/1_wandb.py
index b9f56de..00685b2 100644
--- a/learnStableBaselines/7_wandb/1_wandb.py
+++ b/learnStableBaselines/7_wandb/1_wandb.py
@@ -14,6 +14,7 @@ config = {
 }
 run = wandb.init(
     project="sb3",
+    dir="/home/gauravb/Documents/LearnProgramming/wandbsavedfiles",
     config=config,
     sync_tensorboard=True,  # auto-upload sb3's tensorboard metrics
     monitor_gym=True,  # auto-upload the videos of agents playing the game
@@ -38,4 +39,4 @@ model.learn(
         verbose=2,
     ),
 )
-run.finish()
\ No newline at end of file
+run.finish()
